.github/CODEOWNERS

```diff
@@ -220,6 +220,7 @@ DEPS @brave/deps-reviewers
 # sources.gni - see docs/gni_sources.md
 **/sources.gni @brave/sources-gni-reviewers
 **/*_sources.gni @brave/sources-gni-reviewers
+android/brave_java_sources.gni @brave/Android
```

Comment:
this code owner seems pointless if every person on the android team is in it?

Reply:
I have refined it here https://github.com/brave/security-action/pull/869/files and plan to remove it from codeowner so it will only flag java source files and not deps. Even if every person currently in the @brave/Android team knows what to do here, future hires should not be automatically added to this list so we should have a separate group to handle this I think?